### PR TITLE
Move the po files download to the `make` call (#infra)

### PR DESCRIPTION
**Pull request by jkonecny12**
Right now if user will call:

`make && make rpms`

it will fail because webui needs to have translations and `make rpms` won't create them. To make this more optimized let's download the translations on the `make` call if they don't present already.

To do that use flag file, which will be created when the files are downloaded and removed on clean.

This is alternative solution to https://github.com/rhinstaller/anaconda/pull/3879 .

### CODE REVIEW

**Review by KKoukiou — APPROVED** [review]
I like it more than my solution.

**Comment by jkonecny12** [comment]
**UPDATED:**
* Fix issue that `make release` without previous `make` call did not packaged '.po' files.

**Comment by jkonecny12** [comment]
The code changed, asking for re-review.

File: po/.gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 *.mo
 *.po.new
+.downloaded
```
Comment (VladimirSlavik) [inline po/.gitignore:3]: It's now `.po-downloaded` and one level up.

**Review by VladimirSlavik — COMMENTED** [review]
Almost there! The ignore no longer matches the file location. Otherwise LGTM.

File: po/.gitignore (hunk 1 above)
Comment (jkonecny12) [reply po/.gitignore]: Fixed.

**Comment by jkonecny12** [comment]
Trying to find out why packit created srpm are broken.

**Review by VladimirSlavik — APPROVED** [review]

**Review by KKoukiou — APPROVED** [review]

**Review by KKoukiou — COMMENTED** [review]
It think you can now drop this hack https://github.com/rhinstaller/anaconda/blob/master/anaconda.spec.in#L345

**Comment by jkonecny12** [comment]
ok, if we encounter some issues we can put that back later.

**Comment by jkonecny12** [comment]
**UPDATED:**
* Request localization all the time during the build.

@M4rtinK do you agree with this change?

**Comment by jkonecny12** [comment]
The packit issues seems to be specific to the packit environment and race condition. Will be probably solved on the Packit side anyway because they are moving SRPM creation to the COPR.

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks! :)

**Pull request merged**